### DISCUSSION TITLE
Cannot get email when changed email

### DISCUSSION THREAD

**Discussion by slekstut** [emailjs-com/help#50] [Q&A] [answered]
Hello,

I have setup first time test email, worked perfectly. Then I changed email to my main, deleted test email, but still emails go to the first one testing email. I was gmail. I dont get why its happening. Docs are checked.

![Snip20230912_14](https://github.com/emailjs-com/help/assets/52723739/3e89efcd-3fc9-497b-8162-947a85d95a8b)

![Snip20230912_15](https://github.com/emailjs-com/help/assets/52723739/3a4d3a21-f32b-43db-9dfd-9c7b0625a9f2)

**Comment by xr0master** [comment, marked as answer]
Hi, you can change the recipient address in the template.

**Reply by slekstut** [reply]
thanks you saved my time !
